python/tidy/servo_tidy/licenseck.py

```diff
@@ -139,3 +66,4 @@
     'name = "webrender"',
     'name = "webrender_traits"',
 ]
+# noqa: Indicate to flake8 that we do not want to check indentation here
```

Comment:
We shouldn't need this anymore since we don't have unindented strings inside a list (the old `licenses` variable).